main/settings.py

```diff
@@ -700,6 +712,31 @@ def decode_base64(env_key, fallback_env_key):
 AZURE_OPENAI_KEY = env("AZURE_OPENAI_KEY")
 AZURE_OPENAI_DEPLOYMENT_NAME = env("AZURE_OPENAI_DEPLOYMENT_NAME")
 
+# FIXME: Do not hard-code http protocol
+LOGIN_REDIRECT_URL = f"http://{FRONTEND_URL}/permalink/login-callback"
+LOGOUT_REDIRECT_URL = f"http://{FRONTEND_URL}/permalink/logout-callback"
```

Comment:
Do we need to define this? We have not added any change to go-web-app regarding the SSO login

Reply:
Using `/sso-auth/` for the redirects for now